testing/handlebars/APKBUILD

```diff
@@ -0,0 +1,62 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Contributor: John Boehr <jbboehr@gmail.com>
+# Maintainer: Rafael del Valle <rvalle@privaz.io>
+
+pkgname=handlebars
+pkgver=0.6.3
+pkgrel=0
+pkgdesc="C implementation library of handlebars.js"
+url="https://github.com/jbboehr/handlebars.c"
+license="LGPLv2.1+"
+arch="all"
+
+_gitrev=8ea5f7163281e29e078cedf28801c7fb72493ed5
+_mustachespec_rev="83b0721610a4e11832e83df19c73ace3289972b9"
+_handlebarsspec_rev="e553716c186961477983d20c6e852d9a387f0668"
+
+depends="talloc"
+makedepends="automake autoconf libtool talloc-dev json-c-dev bison flex yaml-dev lmdb-dev pcre-dev check-dev"
+subpackages="$pkgname-dev $pkgname-utils"
+
+source="handlebars.c-${_gitrev}.tar.gz::https://github.com/jbboehr/handlebars.c/archive/${_gitrev}.tar.gz
```

Comment:
Why do you use _gitrev instead of pkgver? https://github.com/jbboehr/handlebars.c/archive/v0.6.3.tar.gz